src/resources/Notifications.js

```diff
@@ -0,0 +1,122 @@
+import Resource from './Resource'
+
+export default class Notifications extends Resource {
+  source = 'notifications';
+
+  /**
+   * List notifications by user
+   *
+   * @param {int} userId - Filter features by userId.
+   * @param {int} start- Starting row for pagination.
+   * @param {int} limit - Limit results per request.
+   * @param {string} [type=''] - Filter results by type.
+   * @param {bool} [unread=''] - Filter results by unread.
+   * @param {bool} [noCache=false] - Flag to turn off the cache.
+   * @return {Object[]} - Notifications.
+   */
+  async list (
+    userId,
+    start,
+    limit,
+    type,
+    unread,
+    noCache = false
+  ) {
+    const params = {
+      userId,
+      start,
+      limit,
+      type,
+      unread,
+      noCache
+    }
+    clean(params)
+    return await this.request({
+      path: `notifications/${userId}`,
+      params
+    })
+  }
+
+  /**
+   * Count notifications by user
+   *
+   * @param {int} userId - Filter features by userId.
+   * @param {int} start- Starting row for pagination.
+   * @param {int} limit - Limit results per request.
+   * @param {string} [type=''] - Filter results by type.
+   * @param {bool} [unread=''] - Filter results by unread.
+   * @param {bool} [noCache=false] - Flag to turn off the cache.
+   * @return {int} - Notifications.
+   */
+  async count (
+    userId,
+    start,
+    limit,
+    type,
+    unread,
+    noCache = false
+  ) {
+    const params = {
+      userId,
+      start,
+      limit,
+      type,
+      unread,
+      noCache
+    }
+    clean(params)
+    const result = await this.request({
+      path: `notifications/count/${userId}`,
+      params
+    })
+    return Array.isArray(result) ? result[0].count : result
+  }
+
+  /**
+   * Create notification
+   *
+   * @param {Object} notification - Notification body.
+   * @return {Object}
+   */
+  async create (notification) {
+    return await this.request({
+      method: 'POST',
+      path: 'notifications',
+      body: notification
+    })
+  }
+
+  /**
+   * Update notification
+   *
+   * @param {Object} notification - Notification body.
+   * @return {Object}
+   */
+  async update (userId, notification) {
+    return await this.request({
+      method: 'PUT',
+      path: `notifications/${userId}`,
```

Comment:
the only update in notification is the read date, so if you send update for certain user and no body it will mark as read all notifications for that user, if you send a body with type on in, it will mark as read all notification for that user and type